core/models/DevtoolsMessagesTable.ts

```diff
@@ -78,9 +78,9 @@ export default class DevtoolsMessagesTable extends SqliteTable<IDevtoolsMessageR
         return `${value.substr(0, 250)}... truncated ${params.payload.length - 250} chars`;
       }
 
-      // clean out post data and headers (we have these in resources table)
+      // clean out post data (we have these in resources table)
       if ((key === 'headers' || key === 'postData') && params.request) {
-        return undefined;
+        return 'SA_REMOVED_FOR_DB';
```

Comment:
What is this string? Should we have a const list somewhere?

Reply:
It's just used to put in the db so you know why it was removed. Didn't seem to deserve a const besides the minor memory saving.